packages/vscode-extension/src/builders/BuildManager.ts

```diff
@@ -73,17 +73,11 @@ export class BuildManager {
         });
         buildResult = await buildIos(
           deviceInfo,
-          getAppRootFolder(),
           forceCleanBuild,
           cancelToken,
           this.buildOutputChannel,
           progressListener,
-          () => {
-            return this.dependencyManager.checkPodsInstalled();
-          },
-          (appRootFolder: string, cleanBuild: boolean, token: CancelToken) => {
-            return this.dependencyManager.installPods(appRootFolder, cleanBuild, token);
-          }
+          this.dependencyManager
```

Comment:
this will conflict with my changes from #587
Maybe you can revert this and changes from `buildIOS` as they seem out of scope of this PR